src/style.css

```diff
@@ -0,0 +1,59 @@
+.body {
+  margin-top: 20px;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search {
+  border: 1px solid rgb(228, 228, 228);
+  border-radius: 3px;
+  width: 100%;
+  padding: 10px 0 10px 62px;
+  background-image: url("images/Search.svg");
```

Comment:
Try to use background from figma, same for all the colors and sizes. This will require of you to put the search icon as an image in label above the input in html.